release_notes/ocp-4-16-release-notes.adoc

```diff
@@ -3335,6 +3335,43 @@ This section will continue to be updated over time to provide notes on enhanceme
 For any {product-title} release, always review the instructions on xref:../updating/updating_a_cluster/updating-cluster-web-console.adoc#updating-cluster-web-console[updating your cluster] properly.
 ====
 
+// 4.16.17
+[id="ocp-4-16-17_{context}"]
+=== RHSA-2024:7944 - {product-title} {product-version}.17 bug fix and security update
+
+Issued: 16 October 2024
+
+{product-title} release {product-version}.17 is now available. The list of bug fixes that are included in the update is documented in the link:https://access.redhat.com/errata/RHSA-2024:7944[RHSA-2024:7944] advisory. The RPM packages that are included in the update are provided by the link:https://access.redhat.com/errata/RHBA-2024:7947[RHBA-2024:7947] advisory.
+
+Space precluded documenting all of the container images for this release in the advisory.
+
+You can view the container images in this release by running the following command:
+
+[source,terminal]
+----
+$ oc adm release info 4.16.17 --pullspecs
+----
+
+[id="ocp-4-16-17-bug-fixes_{context}"]
+==== Bug fixes
+
+* Previously, the Ingress and DNS operators failed to start correctly because of rotating root certificates. With this release, the Ingress and DNS operator Kubeconfigs are conditionally managed by using the annotation that defines when the PKI requires management and the issue is resolved.. (link:https://issues.redhat.com/browse/OCPBUGS-42431[*OCPBUGS-42431*])
```

Comment:
```suggestion
* Previously, the Ingress and DNS Operators failed to start correctly because of rotating root certificates. With this release, the Ingress and DNS Operator kubeconfigs are conditionally managed by using the annotation that defines when the PKI requires management, and the issue is resolved. (link:https://issues.redhat.com/browse/OCPBUGS-42431[*OCPBUGS-42431*])
```

Reply:
good catch! TY!